### PR TITLE
Upgrade to upstream version 1.17.0

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *new version upstream*

## Solution
- *Upgrade to upstream v.1.17.0*
- *Upgrade to PHP7.3*
- *Small typo stuff*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/freshrss_ynh%20PR106%20(ericgaspar)/)

### CODE REVIEW

File: check_process [hunk 1]
```diff
@@ -13,7 +13,7 @@
         setup_private=0
         setup_public=0
 		upgrade=1
-        upgrade=1	from_commit=b89554865459520d7d90a5a666f09f6987f41de6
+        #upgrade=1	from_commit=b89554865459520d7d90a5a666f09f6987f41de6
```
Comment (yalh76) [inline check_process:16]: Why removing test upgrade from previous version?

File: conf/freshrss.cron [hunk 2]
```diff
@@ -1 +1 @@
-*/10 * * * * __APP__ /usr/bin/php7.0 __FINALPATH__/app/actualize_script.php >/tmp/__APP__.log 2>&1
+*/10 * * * * __APP__ /usr/bin/php7.3 __FINALPATH__/app/actualize_script.php >/tmp/__APP__.log 2>&1
```
Comment (yalh76) [inline conf/freshrss.cron:1]: Would be cool to replace php7.3 to php__PHPVERSION__, would be future proof.

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: check_process (hunk 1 above)
Comment (ericgaspar) [reply check_process:16]: Test upgrade from previous version was freezing the CI... putting it back...

Comment (yalh76) [reply check_process]: At least you can test upgrade from current version 0b1c3be0716e4800f2ed6f5b3065f707ed67b6bf

**Review by yalh76 — APPROVED** [review]
Quick review, LGTM

**Pull request merged**